core/00_infra.js

```diff
@@ -33,21 +33,25 @@
   const RING_SIZE = 4 * 1024;
   const NO_PROMISE = null; // Alias to null is faster than plain nulls
   const promiseRing = ArrayPrototypeFill(new Array(RING_SIZE), NO_PROMISE);
-  // TODO(bartlomieju): it future use `v8::Private` so it's not visible
+  // TODO(bartlomieju): in the future use `v8::Private` so it's not visible
   // to users. Currently missing bindings.
```

Comment:
I'm not sure if this comment makes much sense anymore. Using `v8::Private` is only available in native code /shrug